Logger/Employee.cs

```diff
@@ -0,0 +1,44 @@
+﻿
+namespace Logger;
+
+public record Employee : IEntity
+{
+    private FullName _name;
+    public int Eid { get; set; }
```

Comment:
```suggestion
    public int EmployeeId { get; set; }
```
`Eid` seems like a not common shorthand, the full name is more clear